<?xml version="1.0"?>

<launch>

	<rosparam file="$(find drones)/launch/params.yaml" command="load"/>

	<group ns="uav1">
	    <node name="uav_detector" pkg="drones" type="target_detector" output="screen">
	        <remap from="/image" to="/iris_1/camera_uav/image_raw"/>
	        <remap from="/camera_info" to="/iris_1/camera_uav/camera_info"/>
	        <param name="uav_name" type="str" value="iris_1" />
	    </node>
	</group>

	<group ns="uav2">
	    <node name="uav_detector" pkg="drones" type="target_detector" output="screen">
	        <remap from="/image" to="/iris_2/camera_uav/image_raw"/>
	        <remap from="/camera_info" to="/iris_2/camera_uav/camera_info"/>
	        <param name="uav_name" type="str" value="iris_2" />
	    </node>
	</group>

	<group ns="uav3">
	    <node name="uav_detector" pkg="drones" type="target_detector" output="screen">
	        <remap from="/image" to="/iris_3/camera_uav/image_raw"/>
	        <remap from="/camera_info" to="/iris_3/camera_uav/camera_info"/>
	        <param name="uav_name" type="str" value="iris_3" />
	    </node>
	</group>

	<node pkg="drones" type="addMarkers" name="addMarkers" output="screen" />

	<node pkg="plotjuggler" type="PlotJuggler" name="plotjuggler" output="screen" />

</launch>